### ISSUE TITLE
typo in config description of timeout

### ISSUE THREAD

**Issue by webern** [rvaiya/keyd#361]
just reading about your cool project for the first time. maybe there is a typo here:

https://github.com/rvaiya/keyd/blob/6dc2d5c4ea76802fd192b143bdd53b1787fd6deb/docs/keyd.scdoc#L360

Is one of the following correct?

```
*timeout(<action 1>, <timeout>, <action 2>)*
	If the key is held in isolation for more than _<timeout> ms_, activate the second
	action, if the key is held for less than _<timeout> ms_ or another key is struck
	before <timeout> ms expires, execute the first action.
```

or

```
*timeout(<action 1>, <timeout>, <action 2>)*
	If the key is held in isolation for more than _<timeout> ms_, activate the first
	action, if the key is held for less than _<timeout> ms_ or another key is struck
	before <timeout> ms expires, execute the second action.
```

**Comment by rvaiya** [comment]
The first is correct. I'm not sure I see the mistake, since this is currently what is documented in the man page.

**Comment by webern** [comment]
Thanks. I opened a PR since it's the easiest way to show you what I was seeing.

I've been meaning to come back to this project and try to use it to get macOS like copy/cut/paste and line/word jumping. I didn't get too far on the first attempt but will try again at some point.

**Comment by webern** [comment]
Closed by #384
